apps/src/code-studio/pd/application/teacher/AdministratorInformation.jsx

```diff
@@ -4,31 +4,48 @@ import {
   SectionHeaders,
 } from '@cdo/apps/generated/pd/teacherApplicationConstants';
 import {FormGroup} from 'react-bootstrap';
+import PropTypes from 'prop-types';
 import {LabelsContext} from '../../form_components_func/LabeledFormComponent';
 import {FormContext} from '../../form_components_func/FormComponent';
 import {LabeledInput} from '../../form_components_func/labeled/LabeledInput';
 import UsPhoneNumberInput from '../../form_components/UsPhoneNumberInput';
 import {LabeledUsPhoneNumberInput} from '../../form_components_func/labeled/LabeledUsPhoneNumberInput';
 import {isEmail} from '@cdo/apps/util/formatValidation';
+import {useRegionalPartner} from '../../components/useRegionalPartner';
 
 const AdministratorInformation = props => {
+  const {data} = props;
+  const [regionalPartner] = useRegionalPartner(data);
+
+  const formDescriptionWithAdminApprovalRequired =
+    'Please provide information for an Administrator/School Leader (i.e. ' +
+    'Principal, Vice Principal, STEM Program Director, etc) who can\n' +
+    'certify that the course will be offered at your school. Upon your\n' +
+    'submission of this application, we will contact the\n' +
+    'Administrator/School Leader that you listed via email in order to\n' +
+    'obtain their approval. Note that your application cannot be fully\n' +
+    'reviewed until there is approval from your administrator/school\n' +
+    'leader. Therefore, we encourage you to follow up with them directly\n' +
```

Comment:
Nit: Could we have "Administrator" and "School Leader" capitalized here since it's capitalized elsewhere in these paragraphs please

Reply:
Good catch! Updated